### ISSUE TITLE
Making a request with network issues produces an uncatchable promise rejection

### ISSUE THREAD

**Issue by jocull** [Dwolla/dwolla-v2-node#5]
Sample code. You can reproduce this by disabling network adapter.

```
let AccountToken = new Client.Token({
    access_token: config.dwolla.access_token,
    refresh_token: config.dwolla.refresh_token,
});

return AccountToken.get('business-classifications')
    .then(ok => console.log(ok))
    .catch(err => console.error('We got it!', err));
```

Resulting unhandled error:

```
Unhandled rejection FetchError: request to https://api-uat.dwolla.com/business-classifications failed, reason: getaddrinfo EAI_AGAIN api-uat.dwolla.com:443
    at ClientRequest.<anonymous> (/home/jocull/Documents/Projects/Node/my-api/node_modules/node-fetch/index.js:127:11)
    at emitOne (events.js:90:13)
    at ClientRequest.emit (events.js:182:7)
    at TLSSocket.socketErrorListener (_http_client.js:306:9)
    at emitOne (events.js:90:13)
    at TLSSocket.emit (events.js:182:7)
    at connectErrorNT (net.js:1010:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by sausman** [comment]
Hey @jocull,

Thanks for the detailed issue. I just released a new version (1.1.1) that should fix this. Let me know if you run into anything else.

**Comment by jocull** [comment]
Thank you!
